examples/website/i3s-picking/src/components/attributes-panel.tsx

```diff
@@ -0,0 +1,153 @@
+import React from 'react';
+import styled from 'styled-components';
+import {FontAwesomeIcon} from '@fortawesome/react-fontawesome';
+import {faTimes} from '@fortawesome/free-solid-svg-icons';
+import {Flex} from './styles';
+import {IconProp} from '@fortawesome/fontawesome-svg-core';
+
+const Container = styled.div`
+  ${Flex}
+  align-items: center;
+  color: rgba(255, 255, 255, 0.6);
+  background: #0e111a;
+  flex-flow: column;
+  right: 0;
+  top: 15px;
+  right: 15px;
+  width: 320px;
+  height: auto;
+  max-height: 75%;
+  z-index: 16;
+  word-break: break-word;
+  border-radius: 8px;
+
+  @media (max-width: 600px) {
+    margin: 0;
+    top: ${(props) => (props.isControlPanelShown ? '95px' : '10px')};
+    max-height: ${(props) => (props.isControlPanelShown ? 'calc(75% - 95px)' : 'calc(75% - 10px)')};
```

Comment:
`max-height` might ge increased because `ControlPanel` now takes less place.

Reply:
Returned back and increased up to 90%